### DISCUSSION TITLE
how to access data when returned from SendHttpRequest activity?

### DISCUSSION THREAD

**Discussion by zyphertechnologies** [elsa-workflows/elsa-core#2466] [Q&A]
I am using *SendHttpRequest* activity (named as activity1) to call an API that returns json data. here is the expression I am using to view the json data returned. 
```{{Activities.activity1.ResponseContent | json}}``` 
It returns following json data
```
{
  "hodId": 0,
  "id": 1,
  "code": "BR001",
  "desc": "Information Technology"
}
```
but if I try to access **desc** attributes with following expression ```{{Activities.activity1.ResponseContent.desc}}``` it returns *empty* string.

What am I missing?

**Comment by sfmskywalker** [comment]
Hard to say for sure, but I suspect it might be a bug. A few things have changed since the latest release. Perhaps you can try the latest 2.4.x prerelease packages. When you do, you can try this workflow to see how it works (it's based om your description, so should be easily adaptable):

[workflow-definition-210f130d1936454480ea176324dea083.zip](https://github.com/elsa-workflows/elsa-core/files/7584763/workflow-definition-210f130d1936454480ea176324dea083.zip)
